scripts/build-appbuilder.sh

```diff
@@ -3,6 +3,18 @@ SENTRY_CLI="node_modules/.bin/sentry-cli"
 SENTRY_ORG="shapediver"
 SENTRY_PROJECT="app-builder"
 MAIN_TARGET="main"
+SKIP_UNCOMMITTED_CHECK=false
+
+# Parse args
+for arg in "$@"; do
+    echo "arg is $arg"
+
+  case $arg in
+    --uncommitted)
+      SKIP_UNCOMMITTED_CHECK=true
+      shift
+  esac
+done
```

Comment:
@LemToUp in which cases would you want to deploy sth that is not committed? 
@MajorMeerkatThe3rd what do you think about this?